src/arm/linux/aarch64-isa.c

```diff
@@ -151,4 +151,25 @@ void cpuinfo_arm64_linux_decode_isa_from_proc_cpuinfo(
 	if (features & CPUINFO_ARM_LINUX_FEATURE_ASIMDFHM) {
 		isa->fhm = true;
 	}
+
+#if defined(__linux__)
+#include <sys/prctl.h>
+
+#ifndef PR_SVE_GET_VL
+#define PR_SVE_GET_VL 51
+#endif
+
+#ifndef PR_SVE_VL_LEN_MASK
+#define PR_SVE_VL_LEN_MASK 0xffff
+#endif
+
+	int ret = prctl(PR_SVE_GET_VL);
+	if (ret < 0) {
+		cpuinfo_log_error("prctl(PR_SVE_GET_VL) failed");
+		isa->svelen = 0; // Assume no SVE support if the call fails
+	} else {
+		// Mask out the SVE vector length bits
+		isa->svelen = ret & PR_SVE_VL_LEN_MASK;
+	}
+#endif
```

Comment:
```suggestion
#else
        isa->svelen = 0;
#endif
```

Reply:
This change is not required anymore as we have removed the following check - **#if defined(__linux__)** (Not required as the code is compiled for Linux anyways).